src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+
+export type ErrorAction = '' | 'load' | 'add' | 'delete' | 'update';
```

Comment:
I think it's better to create an enum with some text values like 'unable to delete todo' or something similar